backend/types/opentok-jwt-d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'opentok-jwt' {
```

Comment:
Note: this was needed since I'm using the `opentok-jwt` package to generate a project token; however, the package is not TypeScript friendly so this was needed. I also cannot it export with a default; therefore, disabling lint since it needs to be imported like this:
```
import { projectToken } from 'opentok-jwt`
```
therefore not supporting the default export approach